queue_job/jobrunner/channels.py

```diff
@@ -410,11 +419,13 @@ def configure(self, config):
 
         * capacity
         * sequential
+        * subcapacity
         * throttle
         """
         assert self.fullname.endswith(config["name"])
         self.capacity = config.get("capacity", None)
         self.sequential = bool(config.get("sequential", False))
+        self.subcapacity = int(config.get("subcapacity", 0)) or None
```

Comment:
I perceive this `0 or None` falsy coercion as contrived. I'm open to a better idiom.

Reply:
Doing it during configuration parsing is an alternative. Then you can have same thing as for `capacity`.

```suggestion
        self.subcapacity = config.get("subcapacity", None)
```

```diff
--- a/queue_job/jobrunner/channels.py
+++ b/queue_job/jobrunner/channels.py
@@ -897,7 +897,16 @@ class ChannelManager:
                             f"Invalid channel config {config_string}: "
                             f"duplicate key {k}"
                         )
-                    config[k] = v
+                    if k == "subcapacity":
+                        try:
+                            config[k] = int(v)
+                        except Exception as ex:
+                            raise ValueError(
+                                f"Invalid channel config {config_string}: "
+                                f"invalid subcapacity {v}"
+                            ) from ex
+                    else:
+                        config[k] = v
             else:
                 config["capacity"] = 1
             res.append(config)
```